app/src/main/java/com/kickstarter/libs/utils/ApplicationLifecycleUtil.java

```diff
@@ -53,40 +54,68 @@ public void onActivityResumed(final @NonNull Activity activity) {
       // Facebook: logs 'install' and 'app activate' App Events.
       AppEventsLogger.activateApp(activity.getApplication());
 
-      // Refresh the config file
-      this.client.config()
-              .compose(Transformers.pipeApiErrorsTo(this::handleConfigApiError))
-              .compose(Transformers.neverError())
-              .subscribe(c -> {
-                //sync save features flags in the config object
-                if (this.build.isDebug() || Build.isInternal()) {
-                  ConfigExtension.syncUserFeatureFlagsFromPref(c, this.featuresFlagPreference);
-                }
-                this.config.config(c);
-              });
-
-
-      // Refresh the user
-      final String accessToken = this.currentUser.getAccessToken();
-      if (ObjectUtils.isNotNull(accessToken)) {
-        this.client.fetchCurrentUser()
-                .compose(Transformers.neverError())
-                .subscribe(u -> this.currentUser.refresh(u));
-      }
+      refreshConfigFile();
+      refreshUser();
 
       this.isInBackground = false;
     }
   }
 
+  /**
+   * Refresh the config file.
+   */
+  private void refreshConfigFile() {
+    this.client.config()
+      .compose(Transformers.pipeApiErrorsTo(this::handleConfigApiError))
+      .compose(Transformers.neverError())
+      .subscribe(c -> {
+        //sync save features flags in the config object
+        if (this.build.isDebug() || Build.isInternal()) {
+          ConfigExtension.syncUserFeatureFlagsFromPref(c, this.featuresFlagPreference);
+        }
+        this.config.config(c);
+      });
+  }
+
   /**
    * Handles a config API error by logging the user out in the case of a 401. We will interpret
    * 401's on the config request as meaning the user's current access token is no longer valid,
    * as that endpoint should never 401 othewise.
    */
   private void handleConfigApiError(final @NonNull ErrorEnvelope error) {
     if (error.httpCode() == 401) {
-      this.logout.execute();
-      ApplicationUtils.startNewDiscoveryActivity(this.application);
+      forceLogout("config_api_error");
+    }
+  }
+
+  /**
+   * Forces the current user session to be logged out.
+   */
+  private void forceLogout(final @NonNull String context) {
+    this.logout.execute();
+    ApplicationUtils.startNewDiscoveryActivity(this.application);
+    Bundle params = new Bundle();
+    params.putString("location", context);
```

Comment:
Are we adding this param because of an Analytic event?

Reply:
Yes! I added the param to know when we are forcing the logout